### PR TITLE
Feature/rsync arguments

### PR DESCRIPTION
**Pull request by greendeath**
Finished work for passing rsync arguments to cdist type:
Added new parameter: optional_multiple
Updated man.text
Added COPYING info (Please, put your name and year before mine since I didn't create this type)

### CODE REVIEW

File: cdist/conf/type/__rsync/gencode-local [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+#
+# cdist is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with cdist. If not, see <http://www.gnu.org/licenses/>.
+#
+
```
Comment (telmich) [inline cdist/conf/type/__rsync/gencode-local]: Finally! A header! The original author must have completely forgotten this!

File: cdist/conf/type/__rsync/man.text [hunk 2]
```diff
@@ -73,12 +80,18 @@ __rsync otherstuff \
     --destination /usr/local/bin \
     --source "$__type/files/package2"
 
+# Use rsync option --exclude
+__rsync /tmp/testdir \
+    --source /etc \
+    --rsync-opts exclude=sshd_conf
+
```
Comment (telmich) [inline cdist/conf/type/__rsync/man.text]: Maybe add a second example with additional --rsync-opts to make it clear that more than one is supported

File: cdist/conf/type/__rsync/man.text [hunk 3]
```diff
@@ -47,6 +47,13 @@ destination::
 remote-user::
     Use this user instead of the default "root" for rsync operations.
 
+rsync-opts::
+    Use this option to give rsync options with.
+    See rsync(1) for available options.
+    Only "--" options are supported.
+    Write the options without the beginning "--"
+    Can be specified multiple times.
+
```
Comment (telmich) [inline cdist/conf/type/__rsync/man.text]: This parameter should probably be listed below "OPTIONAL MULTIPLE" section

**Comment by telmich** [comment]
Overall good, can be merged after fixing these small problems

File: cdist/conf/type/__rsync/man.text (hunk 3 above)
Comment (greendeath) [inline cdist/conf/type/__rsync/man.text]: I took https://github.com/telmich/cdist/blob/master/cdist/conf/type/__ssh_authorized_key/man.text as a template, but yeah i can add a new section

**Pull request merged**